repl/src/main/scala/ammonite/repl/interp/AmmonitePlugin.scala

```diff
@@ -150,4 +170,49 @@ object AmmonitePlugin{
 
     output(open.toVector)
   }
+}
+
+
+object LineNumberModifier {
+  def apply(g: Global)(unit: g.CompilationUnit,
+                       output: Seq[ImportData] => Unit,
+                       importsLen: => Int) = {
+
+    import g._
+
+    class LineNumberCorrector(unit: CompilationUnit) extends Transformer {
```

Comment:
Since we're only ever instantiating `LineNumberCorrector` once, could we make it an `object` instead of a `class`?